source/_integrations/zha.markdown

```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
 
 There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
 
+## Introduction and dictionary (terminology vocabulary)
+
+ZHA is a native “Zigbee host application” for Home Assistant that via a Zigbee Coordinator adapter will setup and allow control of a single “Zigbee network” (using mesh topology). Once installed ZHA you will be able to directly join/pair most types of “Zigbee Router Device” and “Zigbee End Device” products to that Zigbee network, regardless of product manufacturer and brand.
+
+Zigbee Router Devices will interconnect with other devices to distribute and repeat transmission messages within the Zigbee network mesh, (they will therefor always need be available so are normally mains-powered).  Zigbee End Devices are usually but not always low-powered products such as battery-operated sensors, these will “sleep” most of the time to save power and only “wake” to report in when changes occur, (example sensor state change like temperature or open and close). 
+
+In most common Zigbee setups you will always need to first setup many Zigbee Router Devices to extend range and coverage of the Zigbee network, otherwise signals and/or messages to far way devices might be lost.
+
+#### ZHA and Zigbee limitations
+  - ZHA only support connecting one Zigbee Coordinator adapter / one Zigbee network.
+    - Zigbee devices can only be joined/paired to one Zigbee Coordinator / connected to one Zigbee network.
```

Comment:
This is not an integration limitation.

Reply:
They actually are. ZHA could add support for an additional Zigbee Coordinator adapter and allow setting up a separate network on that, (I read that is something that Zigbee2MQTT is planning to add support for). 

Newer Zigbee Coordinator stacks/firmware from Silicon Labs and does also support "multi-PAN" (multi-network) which allows having more than one Zigbee network on a single  Zigbee Coordinator adapter. 

A real-world scenario example for those is when for example a "production" Zigbee network for stable devices and a "test" network for unstable and troublesome devices. instance. Another example is having older non-Zigbee 3.0 devices on one Zigbee Coordinator and newer Zigbee 3.0 devices on a different Zigbee Coordinator as they have different security, as that way newer Zigbee 3.0 devices do not need to run lower security to be compatible with older devices.

An alternative could be for Home Assistant to allow the installation of multiple instances of the ZHA integration.

You can already achieve something similar with Zigbee2MQTT by adding another remote Zigbee2MQTT.

Reply:
> Zigbee devices can only be joined/paired to one Zigbee Coordinator / connected to one Zigbee network.

This line has nothing to do with the integration. Nothing in this line of documentation indicates that or is related to the integration. You last comment is probably thinking about something else. I'll ignore.